config/prod.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   },
   server: {
     api: {
-      url: 'http://prod--gateway.elife.internal/',
+      url: 'http://prod--cdn-gateway.elifesciences.org/',
```

Comment:
This will be upgrade to HTTPS, but why not https://api.elifesciences.org/?

Though given it was the internal address, this has to be server-side only, and will now be going across the road to Fastly. Assuming these are cacheable requests, probably still worth it (if there's no local cache).

Reply:
I agree on not ideal because it has to go to Fastly rather than staying in the VPC; but simpler than  running a local proxy or implementing a cache.

`prod--cdn-gateway` is consistent by environment (changing `prod`) whereas `api` is specific to `prod`; in fact also `end2end` and `staging` configurations should be changed here to use the same pattern.

Reply:
And yes, has to be HTTPS.